src/query_parser/v2/parser.y

```diff
@@ -278,16 +281,11 @@ static inline char *toksep2(char **s, size_t *tokLen) {
   });
 }
 
-%type modifierlist { Vector* }
+%type modifier { FieldName }
```

Comment:
Is this the largest field in the union? If so, consider changing to `FieldName *` and heap allocating.
See [link](https://sqlite.org/src/doc/trunk/doc/lemon.html#ptype:~:text=But%20the%20grammar%20designer,just%20need%20to%20know) for motivation.

Reply:
This is OK today, as this type is `QueryToken` + a pointer, and we already use larger structs in the union